packages/dai/test/eth/EthereumCdpService.spec.js

```diff
@@ -50,6 +51,21 @@ describe('find cdp', () => {
     expect(sameCdp.dsProxyAddress).not.toBeDefined();
   });
 
+  test('regression: handle null proxy correctly', async () => {
+    const cdps = buildTestEthereumCdpService({
+      accounts: {
+        default: { type: 'privateKey', ...TestAccountProvider.nextAccount() }
+      },
+      proxy: {
```

Comment:
pretty sure these three lines don't do anything

Reply:
each service has its own way of handling the configuration passed to it -- in this case, DSProxyService doesn't do anything with configuration

Reply:
I guess I was too married to my stub.  These three lines didn't even do what I wanted them to as this is just configuration.  It was just a coincidence it worked.  I'll remove the lines.